assets/app/features/upgrade/i18n/en.json

```diff
@@ -23,6 +23,7 @@
                         "updates_installed": "Maintenance Updates to SLES SP1",
                         "network_sanity": "Cloud Health",
                         "high_availability": "High Availability Health",
+                        "storage": "Storage Health",
                         "free_node_available": "Free node available"
```

Comment:
Wrong line, was meant for high_availability but I can see that it has moved to optionalPreChecks so ignore this comment :+1: